### PR TITLE
Add wide dates timeseries output

### PR DESCRIPTION
**Pull request by TomGoBravo**
##  This PR

* Adds a new csv output that has a column for each date. This file is 1/5 the size of the current `timeseries.csv` and includes the provenance and summary data.
* Reverts back to sorted timeseries-summary.csv, as disabled by https://github.com/covid-projections/covid-data-model/pull/606 because we want the file sorted for easier diffing.

I'm not totally sure how I'm going to use `timeseries-wide-dates.csv` :-/ but want to try having it on-hand for debugging data problems.

### Please Check
- [ ] Are the [JSON schemas](https://github.com/covid-projections/covid-data-model/tree/master/api/schemas) updated?
- [ ] Are the [api docs](https://github.com/covid-projections/covid-data-model/blob/master/api/README.V1.md) updated?
- [ ] Are tests passing?

### CODE REVIEW

File: libs/datasets/timeseries.py [hunk 1]
```diff
@@ -96,6 +97,51 @@ def latest_values(self) -> pd.DataFrame:
             .reset_index(drop=True)
         )
 
+    def get_date_columns(self) -> pd.DataFrame:
```
Comment (ghop02) [inline libs/datasets/timeseries.py:100]: Can you add a docstring to this?

File: libs/datasets/timeseries.py [hunk 2]
```diff
@@ -12,6 +12,7 @@
 from libs.datasets.common_fields import CommonIndexFields
 from libs.datasets.common_fields import CommonFields
 from libs.datasets.dataset_utils import AggregationLevel
+from libs.qa.dataset_summary_gen import generate_field_summary
```
Comment (ghop02) [inline libs/datasets/timeseries.py:15]: would prefer to import the module ;)

File: libs/datasets/timeseries.py (hunk 1 above)
Comment (TomGoBravo) [reply libs/datasets/timeseries.py:100]: done

File: libs/datasets/timeseries.py (hunk 2 above)
Comment (TomGoBravo) [reply libs/datasets/timeseries.py]: done

**Pull request merged**